iterfzf/__init__.py

```diff
@@ -33,7 +33,7 @@ def iterfzf(
     multi=False,
     mouse=True,
     print_query=False,
-    cycle=True,
+    cycle=False,
```

Comment:
IMHO the `cycle` option should be turned off by default if `fzf` also turns off the `--cycle` option by default.  Could you squash this commit (1ef6e1358b9a1457a675209109303ddbcae1cd78) into f8ec3eeb5ca0c194fd9c616088e17b2080352ea7?